plugins/node.d/snmp__df.in

```diff
@@ -106,13 +107,15 @@ if (!defined ($session))
 
 my %partitions;
 
-my $parts = get_by_regex ($session, $hrFSIndex, "[1-9]");
+my $filesystems = get_by_regex ($session, $hrFSIndex, "[1-9]");
 
-foreach my $partition (keys %$parts)
+foreach my $filesys (keys %$filesystems)
 {
-    my $mp = get_single ($session, $hrFSMountPoint . $partition);
-    $partitions{$mp}{partition} = $partition;
-    print "# Added partition \"$mp\" as $partition...\n" if $Munin::Plugin::SNMP::DEBUG;
+	my $mp = get_single ($session, $hrFSMountPoint . $filesys);
```

Comment:
Please make sure, that you stick to the currently used whitespace style of this plugin (do not mix spaces and tabs).